x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -44,7 +44,7 @@ interface Props {
   fetchStatus: FETCH_STATUS;
   height?: number;
   onToggleLegend?: LegendItemListener;
-  timeseries: TimeSeries[];
+  timeseries?: TimeSeries[] | TimeSeries;
```

Comment:
Wouldn't it be simpler to leave this as `timeseries: TimeSeries[];` and then require consumers to wrap their series in `[series]` if there's only one? The consumer would know this ahead of time and wouldn't have to dynamically check like here.